packages/social-trader/src/components/notifications/components/notification/notification-helpers.ts

```diff
@@ -0,0 +1,90 @@
+import { NotificationLocationViewModel } from "gv-api-web";
+import NewsIcon from "media/news.svg";
+import RedUserIcon from "media/red-user.svg";
+import { DASHBOARD_ROUTE } from "routes/dashboard.routes";
+import { composePostDetailsUrl, MEDIA_ROUTE } from "routes/social.routes";
+import {
+  composeAccountDetailsUrl,
+  composeFollowDetailsUrl,
+  composeFundsDetailsUrl,
+  composeManagerDetailsUrl,
+  composeProgramDetailsUrl
+} from "utils/compose-url";
+
+import { LINKS, TYPE } from "./notification";
+
+export const getStaticIconUrl = (type: string): string | null => {
+  switch (type) {
+    case TYPE.PROFILE: {
+      return RedUserIcon;
+    }
+    case TYPE.PLATFORM: {
+      return NewsIcon;
+    }
+    default:
+      return null;
+  }
+};
+
+export const getLink = (location: NotificationLocationViewModel): string => {
+  const { id } = location;
+  switch (location?.location) {
+    case LINKS.TRADING_ACCOUNT: {
+      return composeAccountDetailsUrl(id);
+    }
+    case LINKS.DASHBOARD: {
+      return DASHBOARD_ROUTE;
+    }
+    case LINKS.FOLLOW: {
+      return composeFollowDetailsUrl(id);
+    }
+    case LINKS.PROGRAM: {
+      return composeProgramDetailsUrl(id);
+    }
+    case LINKS.SOCIAL_POST: {
+      return composePostDetailsUrl(id);
+    }
+    case LINKS.USER: {
+      return composeManagerDetailsUrl(id);
+    }
+    case LINKS.FUND: {
+      return composeFundsDetailsUrl(id);
+    }
+    case LINKS.SOCIAL_MEDIA_POST: {
+      return MEDIA_ROUTE;
+    }
+    default:
+      return "";
+  }
+};
+
+export const getColor = (type: string): string => {
+  switch (type) {
+    case TYPE.PROFILE: {
+      return "#D6CE93";
```

Comment:
Where you get these colors?..

Reply:
There is no indication in task which colors should i use, so i've decided to use random ones. If you are not happy with this, just tell me which ones should i use.

Reply:
If the task doesn't require changing colors don't change them.

Reply:
GV-3684 requires changing colors

Reply:
If GV-3684 requires changing colors, then ask for specific colors. It doesn't have to be random.